includes/class-jwt-auth-schema-filters.php

```diff
@@ -16,14 +16,90 @@
  * Class JWT_Auth_Schema_Filters
  */
 class JWT_Auth_Schema_Filters {
+	/**
+	 * Adds filters.
+	 *
+	 * @return string|null
+	 */
+	public static function get_auth_class() {
+		if ( class_exists( '\WPGraphQL\JWT_Authentication\Auth' ) ) {
+			return '\WPGraphQL\JWT_Authentication\Auth';
+		} elseif ( class_exists( \WPGraphQL\Login\Main::class ) ) {
+			return \WPGraphQL\Login\Auth\TokenManager::class;
+		} else {
+			return null;
+		}
```

Comment:
1. class_exists() doesn't use a `\` prefix.
2. using the string instead of `::class` for the check, so it doesn't error if the class doesnt exist.
3. Checking for the class that's actually used instead of the parent class.
4. Early return to reduce the code complexity.
 
```suggestion
		if ( class_exists( 'WPGraphQL\JWT_Authentication\Auth' ) ) {
			return \WPGraphQL\JWT_Authentication\Auth::class;
		}

		if ( class_exists( 'WPGraphQL\Login\Auth\TokenManager' ) ) {
			return \WPGraphQL\Login\Auth\TokenManager::class;
		}

		return null;
```